install/linux/usr/share/odemis/hwtest/tmcm-can-ctrl.odm.yaml

```diff
@@ -0,0 +1,20 @@
+SimOptical: {
+    class: Microscope,
+    role: brightfield,
+    children: [Light, TMCM_CAN],
+}
+
+Light: {
+    class: simulated.Light,
+    role: brightlight,
+}
+
+TMCM_CAN: {
+    class: tmcm.CANController,
+    role: null,
```

Comment:
Let's put a role, just for the sake of it. Maybe "rotator" ? Or what role will this hardware have in the Sonic?